superbench/analyzer/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Exposes interfaces of SuperBench Analyzer."""
+
+__all__ = ['data_analysis']
```

Comment:
Looks there have no difference whether put `data_analysis` into `__all__` or not. If you import it in this way:
`import superbench.analyzer.data_analysis as data_analysis `

Reply:
removed this file